devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/java/build.gradle-template.ftl

```diff
@@ -26,9 +14,6 @@ dependencies {
 
     testImplementation 'io.quarkus:quarkus-junit5'
     testImplementation 'io.rest-assured:rest-assured'
-
-    nativeTestImplementation 'io.quarkus:quarkus-junit5'
-    nativeTestImplementation 'io.rest-assured:rest-assured'
```

Comment:
Were these removed because the dependencies are "inherrited" from the test dependencies?

Reply:
Yes